src/utils/identity.js

```diff
@@ -0,0 +1,3 @@
+export default function identity(value) {
+  return value;
+}
```

Comment:
@goatslacker Grouping these in one file sounds like a good idea to me however I worry it doesn't follow the style of the rest of utils of exporting one function per module (or as @taylorhakes mentions inlining some of them, especially simple ones like identity). Also, exporting arrow functions is pleasantly succinct but also another question of style :). @gaearon thoughts?